internal/commands/waypoint/opts/opts.go

```diff
@@ -17,16 +19,18 @@ import (
 
 func New(ctx *cmd.Context) WaypointOpts {
 	return WaypointOpts{
-		Ctx:     ctx.ShutdownCtx,
-		Profile: ctx.Profile,
-		IO:      ctx.IO,
-		Output:  ctx.Output,
-		WS:      waypoint_service.New(ctx.HCP, nil),
+		Ctx:          ctx.ShutdownCtx,
+		Profile:      ctx.Profile,
+		IO:           ctx.IO,
+		Output:       ctx.Output,
+		WS:           waypoint_service.New(ctx.HCP, nil),
```

Comment:
Should this be still creating the WS based on the 20230818 service?

Reply:
That was kept there while the downstream PRs were in-flight so that `hcp` would still compile (if you removed that and things were still using it, you'd have to update all usages right away). Now that they're merged here, I'll remove this!

Reply:
Updated! https://github.com/hashicorp/hcp/pull/208/commits/e2639e10276234736e06503676a2d32a46e15b16